Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/firebase/FirebaseHelper.kt

```diff
@@ -0,0 +1,21 @@
+package com.mentalmachines.droidcon_boston.firebase
+
+
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+
+class FirebaseHelper private constructor() {
+
+    val database: FirebaseDatabase
+    val mainDatabase: DatabaseReference
+
+    init {
+        this.database = FirebaseDatabase.getInstance()
```

Comment:
might be worth flipping on persistence?
https://firebase.google.com/docs/database/android/offline-capabilities

Reply:
done